.mergify.yml

```diff
@@ -130,3 +130,13 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}] {{ title }} (backport #{{ number }})"
+
+  - name: backport patches to 9.1 branch
+    conditions:
+      - merged
+      - base=main
+      - label=backport-9.1
+    actions:
+      backport:
+        branches:
+          - "9.1"
```

Comment:
Not sure how it appeared, I think we need to make sure only jruby lock file is updated with dependency update workflow.
```suggestion
```